MastodonSDK/Sources/MastodonUI/View/Content/StatusCardControl.swift

```diff
@@ -0,0 +1,337 @@
+    private var dividerConstraint: NSLayoutConstraint?
+
+    public override var isHighlighted: Bool {
+        didSet {
+            // override UIKit behavior of highlighting subviews when cell is highlighted
+            if isHighlighted,
+               let cell = sequence(first: self, next: \.superview).first(where: { $0 is UITableViewCell }) as? UITableViewCell {
+                highlightView.isHidden = cell.isHighlighted
+            } else {
+                highlightView.isHidden = !isHighlighted
+            }
+        }
+    }
+
+    public override init(frame: CGRect) {
+        super.init(frame: frame)
+
+        apply(theme: ThemeService.shared.currentTheme.value)
+
+        ThemeService.shared.currentTheme.sink { [weak self] theme in
+            self?.apply(theme: theme)
+        }.store(in: &disposeBag)
+
+        clipsToBounds = true
+        layer.cornerCurve = .continuous
+        layer.cornerRadius = 10
+
+        if #available(iOS 15, *) {
+            maximumContentSizeCategory = .accessibilityLarge
+        }
+
+        highlightView.backgroundColor = UIColor.label.withAlphaComponent(0.1)
+        highlightView.isHidden = true
+
+        titleLabel.numberOfLines = 2
+        titleLabel.textColor = Asset.Colors.Label.primary.color
+        titleLabel.font = .preferredFont(forTextStyle: .body)
+
+        linkLabel.numberOfLines = 1
+        linkLabel.textColor = Asset.Colors.Label.secondary.color
+        linkLabel.font = .preferredFont(forTextStyle: .subheadline)
+
+        imageView.tintColor = Asset.Colors.Label.secondary.color
+        imageView.contentMode = .scaleAspectFill
+        imageView.clipsToBounds = true
+        imageView.setContentHuggingPriority(.zero, for: .horizontal)
+        imageView.setContentHuggingPriority(.zero, for: .vertical)
+        imageView.setContentCompressionResistancePriority(.zero, for: .horizontal)
+        imageView.setContentCompressionResistancePriority(.zero, for: .vertical)
+
+        labelStackView.addArrangedSubview(titleLabel)
+        labelStackView.addArrangedSubview(linkLabel)
+        labelStackView.layoutMargins = .init(top: 10, left: 10, bottom: 10, right: 10)
+        labelStackView.isLayoutMarginsRelativeArrangement = true
+        labelStackView.axis = .vertical
+        labelStackView.spacing = 2
+
+        containerStackView.addArrangedSubview(imageView)
+        containerStackView.addArrangedSubview(dividerView)
+        containerStackView.addArrangedSubview(labelStackView)
+        containerStackView.isUserInteractionEnabled = false
+        containerStackView.distribution = .fill
+
+        addSubview(containerStackView)
+        addSubview(highlightView)
+        addSubview(showEmbedButton)
+
+        containerStackView.translatesAutoresizingMaskIntoConstraints = false
+        highlightView.translatesAutoresizingMaskIntoConstraints = false
+        showEmbedButton.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
There are layout constraint warnings because `dividerView`'s Autoresizing Mask is translated into constraints, fix:

```suggestion
        showEmbedButton.translatesAutoresizingMaskIntoConstraints = false
        dividerView.translatesAutoresizingMaskIntoConstraints = false
```